styles/stylesheet.css

```diff
@@ -0,0 +1,370 @@
+* {
+  box-sizing: border-box;
+}
+html, body {
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+body {
+  min-height: 100%;
+  display: grid;
+  grid-template-rows: 1fr auto;
+  font-family: 'Arial', sans-serif;
+}
+body a {
+  text-decoration: none; /* removes hyperlink underlines */
+}
+body a:hover {
+  color: black;
+  text-decoration: underline;
+}
+/********** MEDIA QUERIES **********/
+/* for phones - dog grid layout needs to change from 3 columns to 1 (home) and hide blog aside */
+@media screen and (max-width: 767.9999px) {
+  .main-content {
+    display: flex;
+    width: 90%;
+    margin: 1rem auto;
+  }
+  .main-content-blog {
+    display: flex;
+    flex-flow: row wrap;
+    margin: 2rem 6rem;
+  }
+  .summary img {
+    display: flex;
+    width: 90%;
+    height: auto;
+  }
+  .summary {
+    width: 90%;
+  }
+  .home-content {
+    display: grid;
+    flex: 70%;
+    align-items: center;
+    margin-bottom: 2rem;
+  }
+  .blog-aside {
+    display: none;
+    flex: 30%;
+    flex-flow: row wrap;
+    margin: -1rem 6rem 2rem 4rem;
+  }
+  .page-content {
+    margin-left: 2rem;
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: 2rem;
+    grid-auto-rows: minmax(20rem, auto);
+  }
+  .page-content-dogs {
+    display: grid;
+    grid-template-columns: repeat(1, 1fr);
+    gap: 3rem;
+    grid-auto-rows: minmax(20rem, auto);
+  }
+  .page-content-dogs section, .page-content section {
+    width: 13rem;
+    border: solid 1px lightgrey;
+    border-radius: 1px;
+  }
+  .page-content-dogs section h3, .page-content-dogs section p, .page-content-dogs section button, .page-content section h3, .page-content section p, .page-content section button {
+    margin: 1rem;
+  }
+  .page-content-dogs section img, .page-content section img {
+    height: 8rem;
+    width: 13rem;
+    padding-right: 1px;
+  }
+  .page-content-dogs section button, .page-content section button {
+    width: 11rem;
+    height: 2rem;
+  }
+}
+/* for tablet - dog grid layout needs to change from 3 columns to 2 (home) */
+@media screen and (min-width: 768px) and (max-width: 991.9999px) {
```

Comment:
This doesn't quite work. I can't see your blog previews at 768. I think your max-width is causing wierdness. Generally you won't need to combine max and min width. You could do a media quiery at 768, then another at 992px just using min-width and as long as the larger value is below the smaller value in css it'll sort itself out.